src/catalog/tests/snapshots/open__initial_audit_log.snap

```diff
@@ -1,5 +1,6 @@
 ---
 source: src/catalog/tests/open.rs
+assertion_line: 444
```

Comment:
@jkosh44, could you tell me what this line is? I think it appeared when I ran `test_persist_open()` / `test_persist_debug()`.

Reply:
Not positive, but it looks like the line in the file `open.rs` that asserts that this snapshot hasn't changed. It's most likely used to help with debugging output and is maybe a new feature of `insta`. This is all speculation, but other `.snap` files have it, so I wouldn't worry about it.

Reply:
Yeah, looks like it's just metadata about this snapshot, https://docs.rs/insta/latest/insta/internals/struct.MetaData.html#method.assertion_line.

Reply:
Ok, thank you!